specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizerReceipt.json

```diff
@@ -87,21 +127,51 @@
+            },
+            "examples": {
+              "application/json": "{\n  \"status\": \"Succeeded\",\n  \"recognitionResults\": [\n    {\n      \"page\": 1,\n      \"clockwiseOrientation\": 1.28,\n      \"width\": 3024,\n      \"height\": 4032,\n      \"unit\": \"pixel\",\n      \"lines\": [\n        {\n          \"boundingBox\": [\n            772,\n            506,\n            1773,\n            517,\n            1771,\n            732,\n            769,\n            721\n          ],\n          \"text\": \"Walmart\",\n          \"words\": [\n            {\n              \"boundingBox\": [\n                786,\n                515,\n                1766,\n                517,\n                1768,\n                736,\n                793,\n                720\n              ],\n              \"text\": \"Walmart\"\n            }\n          ]\n        },\n        ...\n      ]\n    }\n  ],\n  \"understandingResults\": [\n    {\n      \"pages\": [\n        1\n      ],\n      \"fields\": {\n        \"MerchantName\": {\n          \"valueType\": \"stringValue\",\n          \"value\": \"Walmart\",\n          \"text\": \"Walmart\",\n          \"elements\": [\n            {\n              \"$ref\": \"#/recognitionResults/0/lines/0/words/0\"\n            }\n          ]\n        },\n        \"MerchantPhoneNumber\": {\n          \"valueType\": \"stringValue\",\n          \"value\": \"+13605710300\",\n          \"text\": \"( 360 ) 571 - 0300\",\n          \"elements\": [\n            {\n              \"$ref\": \"#/recognitionResults/0/lines/2/words/0\"\n            },\n            {\n              \"$ref\": \"#/recognitionResults/0/lines/2/words/1\"\n            },\n            {\n              \"$ref\": \"#/recognitionResults/0/lines/2/words/2\"\n            },\n            {\n              \"$ref\": \"#/recognitionResults/0/lines/2/words/3\"\n            },\n            {\n              \"$ref\": \"#/recognitionResults/0/lines/2/words/4\"\n            },\n            {\n              \"$ref\": \"#/recognitionResults/0/lines/2/words/5\"\n            }\n          ]\n        },\n        \"TransactionDate\": {\n          \"valueType\": \"stringValue\",\n          \"value\": \"2012-03-06\",\n          \"text\": \"03/06/12\",\n          \"elements\": [\n            {\n              \"$ref\": \"#/recognitionResults/0/lines/31/words/0\"\n            }\n          ]\n        },\n        \"Total\": {\n          \"valueType\": \"numberValue\",\n          \"value\": 18.33,\n          \"text\": \"18.33\",\n          \"elements\": [\n            {\n              \"$ref\": \"#/recognitionResults/0/lines/18/words/0\"\n            }\n          ]\n        },\n        ...\n      }\n    }\n  ]\n}"
             }
           },
-          "default": {
-            "description": "Error response.",
-            "schema": {
-              "$ref": "FormRecognizerReceiptOcr.json#/definitions/ComputerVisionError"
+          "404": {
```

Comment:
I was told we should not define errors in swagger.
Otherwise the generated SDK will not throw exceptions for those errors.

Need ADX team to confirm if it still holds.

Reply:
We do see errors during model validation in the PR due to the error codes.  In particular, it is expecting that we have examples for each response code.  I am okay with removing the error response codes and replacing them with the previous "default".